test/swar/BasicOperations.cpp

```diff
@@ -6,15 +6,146 @@
 #include <type_traits>
 
 using namespace zoo;
+using namespace zoo::swar;
+
+#define HE(nbits, t, v0, v1) static_assert(horizontalEquality<nbits, t>(SWAR<nbits, t>(v0), SWAR<nbits, t>(makeBitmask<nbits, t>(v1))));
+HE(8, u64, 0x0808'0808'0808'0808, 0x8);
```

Comment:
Why is the last element a bit-mask?
This would be a chance to use octal notation for 3 bits

Reply:
It just happened that way (in that I wanted HE to test something function based against something constant, and make bitmask was it.)